src/core/Akka/Actor/RepointableActorRef.cs

```diff
@@ -24,16 +24,16 @@ public class RepointableActorRef : ActorRefWithCell, IRepointableRef
         private readonly ActorSystemImpl _system;
         private readonly Props _props;
         private readonly MessageDispatcher _dispatcher;
-        private readonly Func<Mailbox> _createMailbox;
+        private readonly MailboxType _mailboxType;
```

Comment:
yeah I noticed that earlier too that we had duplicates of these fields in `RouterActorRef`. I'll make this change and consolidate them.